src/components/PageNavLink/PageNavLink.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+type Props = {
+  text: string;
+  to: string;
+};
+
+export const PageNavLink: React.FC<Props> = React.memo(({ text, to }) => (
+  <NavLink
+    to={to}
+    className={({ isActive }) => (
+      classNames(
+        'navbar-item',
+        {
+          'has-background-grey-lighter': isActive,
+        },
+      )
+    )}
+  >
+    {text}
+  </NavLink>
+));
```

Comment:
if I correct, useMemo without dependencies array do nothing with optimization

Reply:
useMemo and React.memo are completely different things - hook and HOC correspondingly